logo/usage_guidelines.md

```diff
@@ -4,21 +4,19 @@ These guidelines provide you with guidance for using the Kubernetes logo. You
 can use the logo on your website or in print without pre-approval, provided you
```

Comment:
we may want to remove this line `You
can use the logo on your website or in print without pre-approval, provided you
follow these basic guidelines.`

Reply:
as it is covered below at the end